packages/devtools_app/lib/src/debugger/hover.dart

```diff
@@ -97,21 +111,20 @@ class HoverCardData {
 class HoverCard {
   HoverCard({
     @required BuildContext context,
-    @required PointerHoverEvent event,
-    @required String title,
     @required Widget contents,
     @required double width,
+    @required Offset position,
+    String title,
   }) {
     final overlayState = Overlay.of(context);
     final colorScheme = Theme.of(context).colorScheme;
     final focusColor = Theme.of(context).focusColor;
     final hoverHeading = colorScheme.hoverTitleTextStyle;
-    final position = event.position;
 
     _overlayEntry = OverlayEntry(builder: (context) {
       return Positioned(
-        left: math.max(0, position.dx - (width / 2.0)),
-        top: position.dy + _hoverYOffset,
+        left: position.dx,
+        top: position.dy,
         child: MouseRegion(
           onExit: (_) {
```

Comment:
Better to use these?

onExit: (_) => remove(),
onEnter: (_) => _hasMouseEntered = true,